.gitattributes

```diff
@@ -20,3 +20,75 @@ SRI-Cambridge/bitfiles/connectal/bluespec_p2/ubuntu18.04_k5.3.0-1028-aws.bkup/si
 GFE/appsBinaries/ota-update-server/FreeBSD/gdb-freebsd-riscv64-static filter=lfs diff=lfs merge=lfs -text
 */appsBinaries/infotainment-server/*/infotainment_server filter=lfs diff=lfs merge=lfs -text
 */appsBinaries/infotainment-server/*/hacked_server filter=lfs diff=lfs merge=lfs -text
+GFE/appsBinaries/ecu-hacks/throttleNominal_ssithInfo.elf filter=lfs diff=lfs merge=lfs -text
```

Comment:
I feel like all these hacks could be wildcarded into one `.gitattributes` line. Other than that, I think this looks fine.